hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/key/OMKeysDeleteRequest.java

```diff
@@ -159,13 +167,15 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager, TermIn
 
         try {
           // check Acl
+          long startNanosDeleteKeysAclCheckLatency = Time.monotonicNowNanos();
           checkKeyAcls(ozoneManager, volumeName, bucketName, keyName,
-              IAccessAuthorizer.ACLType.DELETE, OzoneObj.ResourceType.KEY,
-              volumeOwner);
+                  IAccessAuthorizer.ACLType.DELETE, OzoneObj.ResourceType.KEY,
+                  volumeOwner);
           OzoneFileStatus fileStatus = getOzoneKeyStatus(
               ozoneManager, omMetadataManager, volumeName, bucketName, keyName);
           addKeyToAppropriateList(omKeyInfoList, omKeyInfo, dirList,
               fileStatus);
+          perfMetrics.setDeleteKeysAclCheckLatencyNs(Time.monotonicNowNanos() - startNanosDeleteKeysAclCheckLatency);
```

Comment:
Do you think we should be repositioning the line where we're updating `deleteKeysAclCheckLatencyNs` right after the `checkKeyAcls` call (similar to `deleteKeysResolveBucketLatencyNs`)?